dataprofiler/profilers/float_column_profile.py

```diff
@@ -150,9 +158,11 @@ def _update_precision(self, df_series, prev_dependent_properties,
         :type df_series: pandas.DataFrame
         :return: None
         """
-        self.precision = max(
-            self.precision, self._get_float_precision(df_series)
-        )
+        subset_precision = self._get_float_precision(df_series)
+        if self.precision and subset_precision:
+            self.precision = min(self.precision, subset_precision)
```

Comment:
Is the switch from max to min intentional or an error here?

Reply:
Intentional. We want the lowest common denominator for precision.